client/signup/steps/site-information/index.jsx

```diff
@@ -119,7 +120,7 @@ export class SiteInformation extends Component {
 
 	renderSubmitButton = () => (
 		<Button primary type="submit" onClick={ this.handleSubmit }>
```

Comment:
Seeing as we're ridding ourselves of the text, maybe we could add some text helper attributes on the button?

`title` for the tooltip and `aria-label` for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).

```
<Button title={ this.props.translate( 'Continue' ) } aria-label={ this.props.translate( 'Continue' ) } primary type="submit" onClick={ this.handleSubmit }>
```

Reply:
NIce. Added.